app/api/currentuser/get_group_invitations.go

```diff
@@ -110,10 +110,16 @@ type groupWithApprovals struct {
 func (srv *Service) getGroupInvitations(w http.ResponseWriter, r *http.Request) service.APIError {
 	user := srv.GetUser(r)
 
-	query := srv.GetStore(r).GroupMembershipChanges().
+	query := srv.GetStore(r).
+		Table(`
+			(SELECT group_id, member_id, MAX(at) AS at, action, ANY_VALUE(initiator_id) AS initiator_id
+			   FROM group_membership_changes
+		    GROUP BY group_id, member_id, action
+			) AS latest_group_membership_changes
```

Comment:
Why is it `latest_group_membership_changes`? Yes, it has MAX(at) instead of `at`, but `initator_id` may come from any row.